src/processor/operator/ddl/create_rel_table_group.cpp

```diff
@@ -0,0 +1,28 @@
+#include "processor/operator/ddl/create_rel_table_group.h"
+
+#include "catalog/rel_table_group_schema.h"
+#include "common/string_utils.h"
+
+using namespace kuzu::common;
+using namespace kuzu::catalog;
+
+namespace kuzu {
+namespace processor {
+
+void CreateRelTableGroup::executeDDLInternal() {
+    auto newRelTableGroupID = catalog->addRelTableGroupSchema(*info);
+    auto writeCatalog = catalog->getWriteVersion();
+    auto newRelTableGroupSchema =
+        (RelTableGroupSchema*)writeCatalog->getTableSchema(newRelTableGroupID);
+    for (auto& relTableID : newRelTableGroupSchema->getRelTableIDs()) {
+        auto newRelTableSchema = writeCatalog->getTableSchema(relTableID);
+        relsStatistics->addTableStatistic((RelTableSchema*)newRelTableSchema);
+    }
+}
+
+std::string CreateRelTableGroup::getOutputMsg() {
```

Comment:
Can you add a test to ddloutputmsg test group for createRelTableGroup output message